requirements.txt

```diff
@@ -1,4 +1,4 @@
-requests==2.8.1
+requests>=2.20.0
```

Comment:
should we pin the version of requests?

Reply:
`requests~=2.20` ?
https://www.python.org/dev/peps/pep-0440/#compatible-release

Or were you thinking an explicit version? #91 asks for a relaxation of that kind of thing.

Reply:
They want to embed service-manager in other projects?
a compatible release version sounds like the best thing?

Reply:
I have updated the requirements.txt and setup.py to use `~2.20`.
Please can you re-approve?